docs/charts/bar.md

```diff
@@ -227,7 +227,7 @@ var stackedBar = new Chart(ctx, {
 });
 ```
 
-## Dataset Properties
+## Dataset Properties for Stacked Charts
```

Comment:
Do we really need a section here? we could simply remove that line

Reply:
Removed. Though that revealed another bug with the anchors. I've updated the PR description to describe